src/Components/Admission/index.js

```diff
@@ -74,19 +73,30 @@ export default function Admission() {
 
   const [values, setValues] = useState({
     senderName: "",
-    senderEmail: "",
+    senderEmail: {
```

Comment:
This need not to be in state. It should be a independent object since it is not going to be changed under any circumstance